src/bin/server.cpp

```diff
@@ -1,36 +1,49 @@
-#include <boost/asio/io_service.hpp>
-
-#include <cstdio>
-#include <cstdlib>
-#include <iostream>
+#include "cxxopts.hpp"
 
 #include "hyrise.hpp"
 #include "scheduler/node_queue_scheduler.hpp"
 #include "server/server.hpp"
-#include "utils/load_table.hpp"
+#include "tpch/tpch_table_generator.hpp"
+
+cxxopts::Options get_server_cli_options() {
+  cxxopts::Options cli_options("./hyriseServer", "Starts Hyrise Server in order to accept network requests.");
+
+  // clang-format off
+  cli_options.add_options()
+    ("help", "Display this help and exit")
+    ("p,port", "Specify the port number. 0 means randomly select an available one", cxxopts::value<uint16_t>()->default_value("5432"))  //NOLINT
+    ("generate_tpch", "Generate all TPC-H tables with specified scale factor (1.0 ~ 1GB)", cxxopts::value<float>()->default_value("0"))  //NOLINT
+    ("debug_note", "Send operator runtimes to the client after statement execution", cxxopts::value<bool>()->default_value("false")) // NOLINT
```

Comment:
Seeing what you actually do with the `debug_note`, I don’t think it’s a good name.